### PR TITLE
Local environment with mkcert

### PR DESCRIPTION
**Pull request by jalxes**

### CODE REVIEW

File: local/Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	until docker-compose exec db pg_isready; do echo "Awaiting for Postgres"; sleep 5; done
+
+init-proxy:
+	docker-compose up -d nginx-proxy mkcert
+
+init-app: 
+	docker-compose up -d web app
+	until docker-compose exec --user www-data app php occ status --output=json | grep -e "{\"installed\":true,"; do echo "Awaiting for NextCloud installation"; sleep 10; done
+
+install-apps:
```
Comment (vitormattos) [inline local/Makefile:16]: It looks very specific. This repository is more generic. I suggest removing this part.

File: local/Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	docker-compose exec --user www-data app php occ app:install deck
+	docker-compose exec --user www-data app php occ app:install calendar
+	docker-compose exec --user www-data app php occ app:install contacts
+
+set-configs: 
+	docker-compose exec --user www-data app php occ config:system:set default_locale --value pt_BR
```
Comment (vitormattos) [inline local/Makefile:22]: Rename this command to set-locale and pass a required argument defining the locale to remove the pt_BR hardcode.

File: local/Readme.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Setup NextCloud
```
Comment (vitormattos) [inline local/Readme.md:1]: It is good to move to the doc folder and in the main README of the project to have a summary with links to the md files in the doc folder

File: local/Readme.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (vitormattos) [inline local/Readme.md:2]: This file is in pt_BR then must go to the doc pt_BR

File: local/Readme.md (hunk 3 above)
Comment (jalxes) [reply local/Readme.md]: i don't know about that, I mean, this README is exclusive for the setup on this folder, so i think it is more simple to have it here

File: local/Readme.md (hunk 4 above)
Comment (jalxes) [reply local/Readme.md]: do you think there should have an english version too?

Comment (vitormattos) [reply local/Readme.md]: Yes, the README is in 2 languages.

**Pull request merged**